mathesar/pagination.py

```diff
@@ -3,6 +3,17 @@
 from rest_framework.pagination import LimitOffsetPagination
 from rest_framework.response import Response
 
+from mathesar.models import Table
+from rest_framework.exceptions import NotFound
+
+
+def get_table_or_404(queryset, pk):
```

Comment:
Instead of repeating this code in two places, please create a `mathesar/utils.api.py` file and put this function there. You can import it into both this file and `viewsets.py`. That way, if we ever change this function, we only need to change it in one place.

Reply:
Eventually, I used `from django.shortcuts import get_object_or_404` to replace `queryset.get(id)`